chunky/src/java/se/llbit/chunky/main/Chunky.java

```diff
@@ -51,20 +34,21 @@
 import java.io.FileInputStream;
 import java.io.FileNotFoundException;
 import java.io.IOException;
-import java.net.URL;
-import java.net.URLClassLoader;
 
 /**
  * Chunky is a Minecraft mapping and rendering tool created by
  * Jesper Öqvist (jesper@llbit.se).
- *
+ * <p>
  * <p>Read more about Chunky at http://chunky.llbit.se .
  */
 public class Chunky {
 
-  /** A log receiver suitable for headless rendering. */
+  /**
```

Comment:
Please try to avoid formatting changes (like this and the extra "<p>") above in code not directly related to your changes in the future. It's just a minor issue, but it makes it easier to review the important changes. Don't worry about changing it now though, that would just create unnecessary extra work.